src/main/java/com/ldtteam/structurize/client/gui/WindowTagTool.java

```diff
@@ -16,22 +17,33 @@
 import net.minecraft.core.BlockPos;
 import net.minecraft.world.level.Level;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
+import static com.ldtteam.structurize.api.constants.Constants.GROUNDLEVEL_TAG;
 
 public class WindowTagTool extends AbstractWindowSkeleton
 {
+    /**
+     * List of tag options. Mods can just insert on this in mod constructor.
+     */
+    public static List<String> TAG_OPTIONS = new ArrayList<>();
```

Comment:
I don't like this being a single static list.  The idea I was thinking about earlier was to have an interface method on the blueprint interface (i.e. anchor block where tags are being saved) so that the list could be building-specific.

Default interface implementation in Structurize could return groundlevel/invisible and other global tags.  Deco controller and hut blocks can provide their specific tags.

Reply:
Because deco controllers also share most of those tags (sit/stand) for example

Reply:
They could both call a common utility method then .

Reply:
I'd like to have most of the tag lists in the same place. So the option is either a static list in each of the individual classes we fill from the same location and query. Or we could have a predicate we fill at that place and just put it directly into structurize. I find the latter a bit more simple